core/testsuite/src/test/java/com/blazebit/persistence/testsuite/WindowFunctionTest.java

```diff
@@ -579,6 +575,14 @@ public void testParsedNamedWindowDefinitionWithRange() {
                 .orderByAsc("per.age")
                 ;
 
+        String queryString = criteria.getQueryString();
+
+        assertEquals("SELECT per.age, window_sum(per.age,'ORDER BY',per.age,'ASC NULLS LAST', " +
```

Comment:
Should use `function('window_sum', ...` for eclipselink: https://travis-ci.org/Blazebit/blaze-persistence/jobs/606361914#L1536